packages/block-editor/src/components/media-replace-flow/style.scss

```diff
@@ -15,15 +15,28 @@
 
 .block-editor-media-flow__url-input {
 	padding: 0 15px;
-	max-width: 255px;
+	max-width: 258px;
 	padding-bottom: 10px;
+	border-top: 1px solid $light-gray-800;
+	margin-top: 5px;
+	padding-top: 15px;
 
 	input {
 		max-width: 180px;
 	}
 }
 
+.block-editor-media-replace-flow__image-url-label {
+	color: $dark-gray-300;
+	font-size: $default-font-size;
+	padding-left: 4px;
+	margin-bottom: -8px;
+	display: block;
+}
+
 .block-editor-media-replace-flow__link-viewer {
+	margin-left: -3px;
```

Comment:
We can get rid of this negative margin if we just override the margin on `.block-editor-url-popover__link-viewer-url` to `margin: 0;`.